enterprise/api_client/enterprise_catalog.py

```diff
@@ -26,7 +26,9 @@ class EnterpriseCatalogApiClient(JwtLmsApiClient):
 
     API_BASE_URL = settings.ENTERPRISE_CATALOG_INTERNAL_ROOT_URL + '/api/v1/'
     ENTERPRISE_CATALOG_ENDPOINT = 'enterprise-catalogs'
-    GET_CONTENT_METADATA_ENDPOINT = ENTERPRISE_CATALOG_ENDPOINT + '/{}/get_content_metadata'
+    # enterprise-catalog has a default page_size of 10, bump to 50 here since we may
+    # be requesting 1000s of records
+    GET_CONTENT_METADATA_ENDPOINT = ENTERPRISE_CATALOG_ENDPOINT + '/{}/get_content_metadata/?page_size=50'
```

Comment:
Being an open source solution at some point it would be nice to pull this value from configuration.